### PR TITLE
Improve index rotation/retention upgrade documentation

### PR DESCRIPTION
**Pull request by bernd**
Make sure people keep their old settings so they will be migrated to the database.

Refs https://github.com/Graylog2/graylog2-server/issues/2166

(I have a diff for master as well. Once this is merged I will commit it.)

### CODE REVIEW

File: UPGRADING.asciidoc [hunk 1]
```diff
@@ -107,6 +107,17 @@ This feature has been disabled by default. If you have enabled the feature the c
...
 ### Removed configuration settings
 
+#### Index Retention and Rotation Settings
+
+In 2.0 the index rotation and retention settings have been moved from the Graylog server config file to the database and are
+now configurable via the web interface.
+
+The old settings from the `graylog.conf` or `/etc/graylog/server/server.conf` will be migrated to the database.
+
+**WARNING:** When you upgrade from a 1.x version and you modified any rotation/retention settings, please make sure you **KEEP** your old settings in the config file so the migration process will add your old settings to the database! Otherwise the retention process will use the default settings and might remove a lot of indices.
```
Comment (joschi) [inline UPGRADING.asciidoc]: Maybe use the standard admonition `WARNING:` here?
http://asciidoctor.org/docs/user-manual/#admonition

File: UPGRADING.asciidoc [hunk 2]
```diff
@@ -107,6 +107,17 @@ This feature has been disabled by default. If you have enabled the feature the c
 
 ### Removed configuration settings
 
+#### Index Retention and Rotation Settings
+
+In 2.0 the index rotation and retention settings have been moved from the Graylog server config file to the database and are
```
Comment (joschi) [inline UPGRADING.asciidoc]: "In Graylog 2.0.0 […]"

**Comment by bernd** [comment]
Updated after feedback.

**Comment by joschi** [comment]
LGTM. :+1:

**Pull request merged**